dspace/config/dspace.cfg

```diff
@@ -907,44 +882,15 @@ cc.license.locale = en
 
 ##### Settings for Thumbnail creation #####
 
-# whether to display thumbnails on browse and search results pages (1.2+)
-# If you have customised the Browse columnlist, then you must also
-# include a 'thumbnail' column in your configuration (1.5+)
-# (This configuration is not used by XMLUI.  To show thumbnails in the
-#  XMLUI, you just need to create a theme which displays them)
-webui.browse.thumbnail.show = false
-
-# max dimensions of the browse/search thumbs. Must be <= thumbnail.maxwidth
-# and thumbnail.maxheight. Only need to be set if required to be smaller than
-# dimension of thumbnails generated by mediafilter (1.2+)
-#webui.browse.thumbnail.maxheight = 80
-#webui.browse.thumbnail.maxwidth = 80
-
-# whether to display the thumb against each bitstream (1.2+)
-# (This configuration is not used by XMLUI.  To show thumbnails in the
-#  XMLUI, you just need to create a theme which displays them)
-webui.item.thumbnail.show = true
-
-# where should clicking on a thumbnail from browse/search take the user
-# Only values currently supported are "item" and "bitstream"
-#webui.browse.thumbnail.linkbehaviour = item
-
 # maximum width and height of generated thumbnails
 thumbnail.maxwidth  = 80
 thumbnail.maxheight = 80
 
 # Blur before scaling.  A little blur before scaling does wonders for keeping
-# moire in check.
+# more in check. (Only used by JPEGFilter)
 thumbnail.blurring = true
 
-# High quality scaling option.  Setting to true can dramatically increase
```

Comment:
Still referenced in  JPEGFilter.

Reply:
Good catch as well. I've restored this setting in https://github.com/DSpace/DSpace/pull/3161/commits/a8d99bba15b4df491fda2390fe3611ef2e6336fd